packages/postcss-minify-params/test/index.js

```diff
@@ -183,6 +183,22 @@ test(
   )
 );
 
+test(
+  'should normalize space in custom property values',
+  processCSS(
+    '@supports (--foo:  ){html{background:green}}',
```

Comment:
It should be only one space, i.e. if we have empty values, keep as is, if we have empty, let print empty space